.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules/
 dist/
 riff-raff-artifact/
-.idea
\ No newline at end of file
+.idea
+riff-raff.yaml
```

Comment:
It's not a blocker, but this is an interesting change if we decide to pursue this tactic more generally in the department. 

I have always quite liked the fact that regardless of team/project, you can normally understand how something is deployed simply by searching for this file in the repo. I wonder if we can auto-generate it but maintain a strong convention for how the file is generated.